src/extension/dxf/core/src/main/java/org/geoserver/wfs/response/dxf/AbstractDXFWriter.java

```diff
@@ -195,14 +195,19 @@ protected ReferencedEnvelope getEnvelope(List featureList) {
      * @param e2
      *
      */
-    private ReferencedEnvelope normalizeEnvelope(ReferencedEnvelope e) {
-        if (e != null) {
+    private ReferencedEnvelope normalizeEnvelope(ReferencedEnvelope pEnv) {
+        if (pEnv != null) {
             // if it's empty, get a 1 meter envelope around it
-            if (e.getWidth() == 0 && e.getHeight() == 0) {
-                e.expandBy(1);
+            if (pEnv.getWidth() == 0 && pEnv.getHeight() == 0) {
+                // no features or no geom, enable creation of valid empty file at least
+                pEnv.init(0d, 1d, 0d, 1d);
             }
+        }else{
```

Comment:
Please be mindful of coding conventions, it should be "} else {". I'm not going to demand this change, but it would be nice if you could fix it.